data/http-server-files.ts

```diff
@@ -2,7 +2,7 @@
  * @title HTTP Server: Serving Files
  * @difficulty intermediate
  * @tags cli, deploy
- * @run --allow-net --allow-read <url>
+ * @run --allow-net --allow-read=static/ <url>
```

Comment:
```suggestion
 * @run --allow-net --allow-read=static/,path/to/file.txt <url>
```

Reply:
@hashrock Like this maybe?

Reply:
yeah, and `static/` -> `public/` because of fsRoot option